conf/systemd.service

```
@@ -2,13 +2,13 @@
 Description=Garage: Data Store
 After=network-online.target
 Wants=network-online.target
+Requires=data.mount
+Requires=metadata.mount
```

Eeeeh we probably want these to be named something more specific than just "data.mount" like idk:

```suggestion
Requires=__APP___data.mount
Requires=__APP___metadata.mount
```

(ie it would be `garage_data.mount`)

---

done